docs/page-markup.md

```diff
@@ -51,5 +51,38 @@ Thus, notice that:
 
 ## Microdata markup
 
-Microdata format support is coming soon.
+SWG uses schema.org markup. Using Microdata, the mark up could look like this:
 
+```
+<div itemscope itemtype="http://schema.org/NewsArticle http://schema.org/Other" id="top">
```

Comment:
Let's remove "http://schema.org/Other" and `id="top"` here.

Reply:
Done.